package.json

```diff
@@ -1,7 +1,11 @@
 {
   "devDependencies": {
+    "@commitlint/cli": "^7",
+    "@commitlint/config-conventional": "^7",
     "@salesforce/dev-config": "1.4.1",
     "ajv": "6.1.1",
+    "commitizen": "^3.0.5",
```

Comment:
Versions are based on what the CLI is using:
    [commitlint/cli](https://github.com/forcedotcom/sfdx-core/blob/3ef407b667cdd65dc2bbd8259924e0b8b7ecb284/yarn.lock#L99) 
    [commitlint/config-conventional](https://github.com/forcedotcom/sfdx-core/blob/3ef407b667cdd65dc2bbd8259924e0b8b7ecb284/yarn.lock#L116)
    [commitizen](https://github.com/forcedotcom/sfdx-core/blob/3ef407b667cdd65dc2bbd8259924e0b8b7ecb284/yarn.lock#L1444)
    [cz-conventional-changelog](https://github.com/forcedotcom/sfdx-core/blob/3ef407b667cdd65dc2bbd8259924e0b8b7ecb284/yarn.lock#L1450)